swagconnect/bitbucket/client.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+import requests
+
+
+class BaseBitbucket:
```

Comment:
Move the functionality from this class to the `BitbucketAPIConnector` class, as they are almost the same